src/uct/sm/mm/sysv/mm_sysv.c

```diff
@@ -195,7 +203,7 @@ static uct_mm_md_mapper_ops_t uct_sysv_md_ops = {
         .mem_free               = uct_sysv_mem_free,
         .mem_advise             = ucs_empty_function_return_unsupported,
         .mem_reg                = ucs_empty_function_return_unsupported,
-        .mem_dereg              = ucs_empty_function_return_unsupported,
+        .mem_dereg              = uct_sysv_mem_dereg,
```

Comment:
why changed?

Reply:
same as for posix: because it should check for INVALIDATE flag - in case if user passed INVALIDATE flag and function return "success" user may expect that callback will be called